core/bin/external_node/src/config/tests/config.yaml

```diff
@@ -0,0 +1,129 @@
+# Sample config parsed by the EN.
+prometheus:
+  listener_port: 3322
+  pushgateway_url: http://prometheus/
+  push_interval_ms: 150
+observability:
+  log_format: json
+  log_directives: warn,zksync=info
+  sentry_url: https://example.com/new
+  sentry_environment: 'mainnet - mainnet2'
+
+postgres:
+  server_url: postgres://postgres:notsecurepassword@localhost:5432/en
```

Comment:
It looks like a mix between secrets and general config, let's split it

Reply:
That's a test configuration though; what purpose would a split serve other than having to manage more files around?